compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtInterfaces.td

```diff
@@ -113,16 +113,7 @@ def LinalgFusionInterface : OpInterface<"LinalgFusionOpInterface"> {
       /*retTy=*/"SmallVector<int64_t, 4>",
       /*methodName=*/"getStaticLoopRanges",
       /*args=*/(ins),
-      /*methodBody=*/"",
-      /*defaultImplementation=*/[{
-          SmallVector<int64_t, 4> loopRanges;
-          llvm::for_each($_op.getOperands(), [&](Value operand) {
-          if (auto shapedType = dyn_cast<ShapedType>(operand.getType())) {
-            llvm::append_range(loopRanges, shapedType.getShape());
-            }
-          });
-          return loopRanges;
-      }]
+      /*methodBody=*/""
```

Comment:
We maybe want to keep the default erroring out as non-implemented. Can we change the return type to `FailureOr<SmallVector<int64_t>>`  and return failure on default?

Reply:
That's a good idea, i'll do that.

Reply:
done